src/ontology/mondo-edit.obo

```diff
@@ -546441,6 +546440,18 @@ intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.o
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8100" xsd:anyURI
 
+[Term]
+id: MONDO:0700282
+name: POLR3-related leukodystrophy
+def: "Hypomyelinating leukodystrophy disorder in which is caused of the disease is a variation in any of the genes encoding POLR3 (RNA polymerase III) subunits, including POLR3A, POLR3B and POLR1C. This disorder is characterized by the association of dental abnormalities (delayed dentition, abnormal order of dentition, hypodontia), hypogonadotropic hypogonadism, and hypomyelinating leukodystrophy manifesting with neurodevelopmental delay or regression and/or progressive cerebellar symptoms." [https://orcid.org/0000-0002-4142-7153, Orphanet:289494, PMID:22855961]
+subset: ordo_disorder {source="Orphanet:289494"}
```

Comment:
note: this was manually added though this subset annotation would be automatically added with the subset update workflow. I added manually so we have that information for this upcoming release (the subset update has already ran)

Reply:
Perfect! Thank you.